packages/react/README.md

```diff
@@ -50,7 +50,7 @@ Documentation lives in our [Storybook](https://smartprocure.github.io/contexture
 
 - [Theming documentation](https://smartprocure.github.io/contexture-react/?path=/story/developer-notes-docs--theme-api)
 - A [Live Demo](https://smartprocure.github.io/contexture-react/?path=/story/live-demos-imdb-search--dynamic-search-layout) using IMDB data
```

Comment:
These are interesting - how are we going to handle per package sites? I guess we can try to put it all in one place in one large site, but storybook is really kind of its own thing 🤔

Reply:
Worst case, we can deploy storybook under a path in the docs site, like `https://smartprocure.github.io/contexture/react/storybook`. But perhaps we can have a different site for it. Ex: https://stellarhoof.github.io/macro-planning and https://stellarhoof.github.io/notes

Reply:
https://github.com/github/pages-gem/issues/724